Content.Shared/Humanoid/HumanoidVisualLayers.cs

```diff
@@ -3,6 +3,10 @@
 
 namespace Content.Shared.Humanoid
 {
+    /// <summary>
+    ///  These are the layer defines for the humanoid sprite system.
+    ///  If you want to add a new layer slot to species? Scroll down~
+    /// </summary>
```

Comment:
Not sure that it's worth having this and the comment below that it references included but up to you.